.github/workflows/ci.yaml

```diff
@@ -10,22 +10,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        package: [
-          'auto-label',
-          'blunderbuss',
-          'buildcop',
-          'conventional-commit-lint',
-          'failurechecker',
-          'gcf-utils',
-          'generate-bot',
-          'header-checker-lint',
-          'label-sync',
-          'merge-on-green',
-          'publish',
-          'release-please',
-          'sync-repo-settings',
-          'trusted-contribution'
-        ]
+        package:
```

Comment:
Did you use an auto-formatter or something for this?

Reply:
Ugh thanks for the catch. I just installed a formatter for YAML in VsCode and forgot to disable.